readthedocs/builds/models.py

```diff
@@ -706,6 +706,13 @@ class Build(models.Model):
         null=True,
         blank=True,
     )
+    build_config_file = models.CharField(
+        _("Non-default build configuration file used"),
```

Comment:
This currently archives the configuration file that was used for the build. So it's mostly an after-the-fact piece of data. It even gets ignored during the build process because I didn't want to add more functionality than the minimal scope.

Reply:
Edit: Now it gets ignored, was actually reading it before.. but that behavior is intended to be deactivated for now.

All I want here is to save the `build_config_file` that was used on a build.

Reply:
> What is the thought behind saving this path and how we think we are going to use it?

The content of the build config file is in Git, as with the whole build. But the _path_ to that config is configuration data that we need to store for reproducibility. Otherwise you won't know where to find it when checking out the build from Git.

Re: naming - right right, I can change the name to `readthedocs_config_file`. I personally found `readthedocs` as pretty implicit in all of the codebase. But who knows, maybe some day the config file has more than just "build" config, even though I'd say that's what it does.

Reply:
`readthedocs_yaml_path` is pretty explicit, IMO. Easy to read, easy to parse, easy to know what we are talking about.